pkg/operator/storageclasscontroller/vmware.go

```diff
@@ -274,6 +277,10 @@ func (v *vCenterAPI) createStorageProfile(ctx context.Context) error {
 	return nil
 }
 
+func (v *vCenterAPI) close(ctx context.Context) error {
+	return v.connection.client.Logout(ctx)
+}
+
```

Comment:
do we need to also logout from restclient? something like `v.connection.restClient.Logout` ?

Reply:
That didn't improve the situation so I decided to leave it out of this PR.

Reply:
> That didn't improve the situation so I decided to leave it out of this PR.

Is this because we did not measure rest client connections or because logging out from SOAP connection also closes REST connection? Is there a downside of logging from restclient?